src/lib/holocene/pagination.svelte

```diff
@@ -143,7 +143,7 @@
       style={floatStyle}
       bind:clientHeight={height}
       class="flex flex-col items-end gap-4 md:flex-row min-w-fit"
-      aria-label={$$restProps['aria-label']}
+      aria-label="{$$restProps['aria-label']} 1"
```

Comment:
I just added a `1` and a `2` here to delineate pagination navigation, but wondering if there might be a better solution.

Reply:
How about "top" and "bottom"?